lib/services/post_service.dart

```diff
@@ -87,10 +97,43 @@ class PostService {
     _postStreamController.add(_posts);
   }
 
-  /// This function is used to add Like to the Post.
+  /// Method to refresh feed of current selected organisation.
+  ///
+  /// **params**:
+  ///   None
   ///
-  /// params:
-  /// * [postId] : id of the post where like need to be added.
+  /// **returns**:
+  /// * `Future<void>`: define_the_return
+  Future<void> refreshFeed() async {
+    _posts.clear();
+    _renderedPostID.clear();
+    await getPosts();
+  }
+
+  ///Method to add newly created post at the very top of the feed.
+  ///
+  /// **params**:
+  /// * `newPost`: define_the_param
```

Comment:
Please explicitly mention the `parameters` & `return types` every where in the docs, As it is a good practice.

Reply:
Hey Azad thanks for mentioning but I'm still working on this I will cover up all the things, proper formatting, proper docs,  proper visibility of widgets when themes are switched, and proper app localization. I need some time bro plus we need to finalize the UI too waiting for the actual volunteers to review this and share their views, so yeah give me some time....